unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -330,6 +344,8 @@ public override void AttachTo(ISharedComponent component)
             SetPlayStateDirty();
             attachedMaterials.Add(component.id, component);
 
+            component.OnAttach -= SetPlayStateDirty;
+            component.OnDetach -= SetPlayStateDirty;
```

Comment:
this also looks a little bit like a hack. Because, as I understand, this component, if it is already subscribed, should be inside `attachedMaterials` and be rejected to attach due to the line 339. Maybe solution could be of movind line 345 to line 341. Also, unsubscritption is not happening for the remained components so maybe it makes sense to Dettach from all components on Disposale...
In any case, can you please try to remove this subsc/unsubsc and check if everything is still working correctly